### ISSUE TITLE
⚠️ Features broken on new issues (beta)

### ISSUE THREAD

**Issue by fregante** [refined-github/refined-github#7856]
### Description

I'm sure every `isConversationList` feature is currently broken on that page. 

### How to replicate the issue + URL

Test URLs:

- https://github.com/download-directory/download-directory.github.io/issues/141
- https://github.com/eslint/eslint/issues
- https://github.com/tc39/proposal-await.ops/issues

The beta is enabled by organization. These three have it. You can spam issues on https://github.com/download-directory/download-directory.github.io/issues if necessary, I'll delete them once we're done. Or leave comments in https://github.com/download-directory/download-directory.github.io/issues/141, better.

**Comment by fregante** [comment]
By the way, finally we have an autocomplete!

<img width="625" alt="Screenshot 3" src="https://github.com/user-attachments/assets/ca369f13-e87d-437b-8c25-064dbfc94615">

**Comment by fregante** [comment]
More features are broken, like the label alignment

**Comment by fregante** [comment]
Bye `quick-comment-hiding`

<img width="500" alt="Screenshot 28" src="https://github.com/user-attachments/assets/994afdf5-2504-4986-91c8-4e28f0bd1a55">

**Comment by fregante** [comment]
These are features running on `isIssueList` and `isSingleIssue`

I'll check these off after verifying them, or with 👋 if they're no longer needed


### Important features


- [ ] https://github.com/refined-github/refined-github/issues/8226
- [x] `comments-time-machine-links`
- [x] https://github.com/refined-github/refined-github/issues/8586
- [x] https://github.com/refined-github/refined-github/issues/8024
- [x] #7968 


### Done

Edit February 6th: Some of these have been broken again. Please open a new issue in that case.

- [x] `quick-mention`
- [x] `selection-in-new-tab`
- [x] `shorten-links`
- [x] `linkify-user-location`
- [x] `readable-title-change-events`
- [x] `jump-to-conversation-close-event`
- [x] `show-names`
- [x] `small-user-avatars`
- [x] `sticky-sidebar`
- [x] `table-input`
- [x] `reactions-avatars`
- [x] `scrollable-areas`
- [x] `prevent-link-loss`
- [x] `collapsible-content-button`
- [x] `embed-gist-inline`
- [x] `emphasize-draft-pr-label`
- [x] `linkify-user-labels`
- [x] `jump-to-conversation-close-event`
- [ ] `avoid-accidental-submissions` 👋
- [ ] `esc-to-cancel` 👋
- [ ] `clean-conversation-headers` 👋
- [ ] `prevent-comment-loss` 👋
- [ ] `quick-comment-hiding` 👋
- [ ] `linkify-user-edit-history-popup` 👋
- [x] bugs-tab
- [x] ci-link
- [x] clean-repo-tabs
- [x] clean-rich-text-editor
- [x] close-out-of-view-modals
- [x] command-palette-navigation-shortcuts
- [x] copy-on-y
- [x] esc-to-deselect-line
- [x] fit-textareas
- [x] hide-diff-signs
- [x] hide-user-forks
- [x] improve-shortcut-help
- [x] linkify-branch-references
- [x] linkify-code
- [x] linkify-line-numbers
- [x] more-dropdown-links
- [x] one-key-formatting
- [x] parse-backticks
- [x] profile-hotkey
- [x] quick-comment-edit
- [x] quick-new-issue
- [x] releases-tab
- [x] reload-failed-proxied-images
- [x] repo-avatars
- [x] repo-header-info
- [x] repo-wide-file-finder
- [x] show-whitespace
- [x] sort-conversations-by-update-time
- [x] tab-to-indent
- [x] unfinished-comments
- [x] user-local-time

**Comment by kovsu** [comment]
| feature  | test url | screenshort | note |
| ------------- | ------------- | ------------- | --------------------------  |
| `avoid-accidental-submissions`  | https://github.com/eslint/eslint/issues/new?q=sort%3Aupdated-desc+is%3Aissue+is%3Aopen&template=bug-report.yml | <img width="200" alt="image" src="https://github.com/user-attachments/assets/076041e2-5a3a-4c98-91ea-0e6226c27385"> |  It doesn't need in `isNewIssue`  |

**Comment by fregante** [comment]
The list above has already been verified. Unless you see that the feature is no longer needed, you can assume they're not working. That would save time from having to build that table

**Comment by kovsu** [comment]
Here has two avatar.

https://github.com/download-directory/download-directory.github.io/issues?q=is%3Apr

<img width="1361" alt="image" src="https://github.com/user-attachments/assets/e9272b20-7a3d-41e1-a476-2d01adc5b102">

**Comment by fregante** [comment]
cc @ToBinio https://github.com/refined-github/refined-github/pull/7924

**Comment by ToBinio** [comment]
@fregante could it be that GitHub is rolling out that feature?
cause for `is:pr` it also renders images without `refined Github`

should I open a PR and just fix it for `is:pr` or do you think it will come for normal issues too and the feature should just be disabled for the new view

**Comment by fregante** [comment]
Uh, they did 😅

**Comment by fregante** [comment]
> * [`preview-hidden-comments`](https://github.com/refined-github/refined-github/blob/main/source/features/preview-hidden-comments.tsx)

We should hold off on implementing this one, their header layout is messy on mobile and I expect them to change it completely. Only fix it if it's a simple class change

**Comment by fregante** [comment]
For advanced features like `close-as-unplanned`, we have to use the UI on behalf of the user because we no longer have the good ol' `form` element 😒 

For example in https://github.com/refined-github/refined-github/pull/7893 this meant opening the dropdown menu and simulating a click on the Edit item.

Here I'd expect the dropdown arrow to be hidden and a new custom button to be added. When the button is clicked, we show the arrow, open it, click "close as not planned", then click the new "close" button. 

<img width="384" alt="Screenshot" src="https://github.com/user-attachments/assets/ac4ecc7e-5bed-40a1-ae98-69cff08bbaf3">

I wonder if it's just easier to do it via API and call `triggerConversationUpdate` (which is also likely broken on the new issue view), but I think that as long as every part here is `select`able, it's actually not too difficult:

https://github.com/refined-github/refined-github/blob/60217e812c4bac4707d1f1b6caea6fd10a0d672e/source/features/quick-comment-edit.tsx#L25-L33

**Comment by darkred** [comment]
Today's blog post announces the gradual rollout of the new issue view to everyone, with all users expected to have access by the end of this week:

https://github.blog/changelog/2025-01-13-evolving-github-issues-public-preview/

> Following our [opt-in preview last year](https://github.blog/changelog/2024-10-01-evolving-github-issues-public-beta/), we are excited to release sub-issues, issue types and advanced search for issues to everyone! 🎉  

> Thank you to everyone who opted-in and gave us feedback on these new additions. We will be rolling out these changes incrementally and expect all users to have access by the end of this week.

**Comment by fregante** [comment]
Oof. We still have a few broken important features. It would be good to address them this week so that it's easy to preserve GHE compat. If not, PRs are not affected yet so that view can be used as "old" for now.

@kovsu do you see anything that you can address? I think the "activity filter" and "clean sidebar" are the most pressing. Others might need a "simple" selector update.

**Comment by kidonng** [comment]
`table-input` needs CSS update:

![Image](https://github.com/user-attachments/assets/2173b67a-9064-4aca-ad4d-a42e78c0debc)

**Comment by injust** [comment]
I think `comments-time-machine-links` might need changes for the new issues.

If I comment this:

```
@injust testing
```

And then Quote reply, I get this:

```
> [@injust](https://github.com/injust?rgh-link-date=2025-01-17T10%3A58%3A26.000Z) testing
```

I don't think the `rgh-link-date` parameter is supposed to appear there, right?

**Comment by mathomp4** [comment]
I was going to make a "The new Bug issue type doesn't trigger bugs-tab" issue, but I suppose that would be this issue? Unless you'd prefer a separate issue for tracking purposes?

**Comment by fregante** [comment]
New issue

**Comment by kovsu** [comment]
`quick-mention` needs to be updated. It doesn't work now.

**Comment by Xelbayria** [comment]
Has the broken "collapsible-content-button" been resolved? I can't use it as is.

**Comment by kovsu** [comment]
@Xelbayria It's working. If you still have any problem please open an issue to provide more details.

![Image](https://github.com/user-attachments/assets/e131a585-1cfd-4605-a659-1c684711196f)

**Comment by SunsetTechuila** [comment]
note on `preview-hidden-comments`: comment body (`'[class^="IssueCommentViewer-module__IssueCommentBody"]'`) is removed from the DOM when the comment is hidden, so we have to either get the data from the react props or by using api

**Comment by fregante** [comment]
If it's missing from the DOM, let's hold off on fixing that feature. Most likely we'll just have to drop it once every view is broken (the feature still works on PRs for now)

**Comment by SunsetTechuila** [comment]
>  we'll just have to drop it

are you against of extracting data from react props?

**Comment by fregante** [comment]
I think we do it in a couple of features but those props are not reliable, meaning they're not **always** there. I don't want bug reports for a feature that works half the time. 

As for calling the API… we could do it, but we have to see if it's worth it. So far nobody even reported that it's been broken.

**Comment by SunsetTechuila** [comment]
> those props are not reliable

maybe this issue is not the place for this discussion, but how are they less reliable than dom elements? also, it seems that with github updates, relying on dom will become less and less feasible

**Comment by fregante** [comment]
DOM is "unreliable" in a different way: always there today, but tomorrow it may change.

React props, if we're talking about the same thing, suffer from the same problem AND one more: they're already **not always there.** Meaning: GitHub may inject those on a page reload, but not when navigating via Ajax.

**Comment by SunsetTechuila** [comment]
does a pr that uses react props have a chance to be merged if it works?

**Comment by fregante** [comment]
There are more useful features that I'd rather see, like #8226 or even the similar  `expand-all-hidden-comments`

**Comment by 0xdevalias** [comment]
I'm not sure if this is a `refined-github` bug, or just a GitHub bug (and haven't got the capacity to check deeper at the moment, so wanted to note it here in case), but today I tried to use the 'Close as duplicate' on an issue I opened, but a repo I don't have permissions on.

<img width="464" height="239" alt="Image" src="https://github.com/user-attachments/assets/fc9e9ddd-1cae-4503-8ace-8a996a57cfd8" />

I didn't end up selecting another issue to link it to as it was on another repo.

It looked like everything worked and closed the issue properly; but then when I did a full refresh / reload / checked the issue list later; I realised that it didn't actually close the issue at all.

---

**Edit:** I just tried to replicate this now and I couldn't.. and checking a different issue it did actually close it as a duplicate properly.. so maybe it was my error.

**Comment by SunsetTechuila** [comment]
> also, it seems that with github updates, relying on dom will become less and less feasible

The same is true for reusing existing elements. Most of the time, we can simply clone and edit them, but not always. Plus, this could be tedious, so I would like to link my comment here: https://github.com/refined-github/refined-github/pull/8712#discussion_r2499138687

Writing this more for future reference than to discuss

**Comment by SunsetTechuila** [comment]
Marked `linkify-user-edit-history-popup` as no longer needed

<img width="547" height="208" alt="Image" src="https://github.com/user-attachments/assets/3683ca49-e3b6-4d1c-bca6-8909d6620eb4" />

**Comment by fregante** [comment]
Loads of features have been restored! I opened individual issues for features that are still broken. Thanks everyone for lending a hand! 🤗
